packages/trading/src/consts.ts

```diff
@@ -1,5 +1,10 @@
 import { EcdsaSigningScheme, SigningScheme } from '@cowprotocol/sdk-order-book'
-import { mapSupportedNetworks, SupportedChainId } from '@cowprotocol/sdk-config'
+import { CowEnv, mapSupportedNetworks, SupportedChainId } from '@cowprotocol/sdk-config'
+
+export const BFF_ENDPOINTS: Record<CowEnv, string> = {
+  prod: 'https://bff.cow.fi',
+  staging: 'https://bff.barn.cow.fi',
```

Comment:
Should we expose this, though?

We could have it as an env var, or accept only the callback that calculates the volatility.
This endpoint uses behind the scenes our Coingecko private API that we don't want to handle out to everyone using the SDK.
If they run their own BFF (like AAVE), they can provide their BFF endpoint.
But to be more user friendly we can ask only for the callback that takes the chain and the tokens, and returns the suggested slippage for them.

Reply:
100% makes sense, thanks!
https://github.com/cowprotocol/cow-sdk/pull/546/commits/75ff23bea0091a5493180b7832fb73f481c73a8b